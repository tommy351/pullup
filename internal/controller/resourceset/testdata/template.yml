---
apiVersion: pullup.dev/v1alpha1
kind: ResourceSet
metadata:
  name: test-46
  namespace: test
spec:
  base:
    ref: master
    sha: efef1cefcd199ce6e0f1f0b9f8acac420563044b
  head:
    ref: fix-test
    sha: aad2b388e813f3a3c90672332bdd2f9c5bac568c
  number: 46
  resources:
    - apiVersion: v1
      kind: Pod
      metadata:
        name: test
      spec:
        containers:
          - name: '{{ .Name }}'
            image: 'gcr.io/test/foo:{{ .Spec.Head.SHA }}'
            env:
              - name: PR_NUMBER
                value: '{{ .Spec.Number }}'
