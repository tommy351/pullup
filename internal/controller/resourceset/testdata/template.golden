- apiVersion: v1
  kind: Pod
  metadata:
    creationTimestamp: null
    name: test-46
    namespace: test
    ownerReferences:
    - apiVersion: pullup.dev/v1alpha1
      blockOwnerDeletion: true
      controller: true
      kind: ResourceSet
      name: test-46
      uid: ""
    selfLink: /api/v1/namespaces/test/pods/test-46
  spec:
    containers:
    - env:
      - name: PR_NUMBER
        value: "46"
      image: gcr.io/test/foo:aad2b388e813f3a3c90672332bdd2f9c5bac568c
      imagePullPolicy: IfNotPresent
      name: test-46
      resources: {}
      terminationMessagePath: /dev/termination-log
      terminationMessagePolicy: File
    dnsPolicy: ClusterFirst
    enableServiceLinks: true
    priority: 0
    restartPolicy: Always
    schedulerName: default-scheduler
    securityContext: {}
    serviceAccount: default
    serviceAccountName: default
    terminationGracePeriodSeconds: 30
    tolerations:
    - effect: NoExecute
      key: node.kubernetes.io/not-ready
      operator: Exists
      tolerationSeconds: 300
    - effect: NoExecute
      key: node.kubernetes.io/unreachable
      operator: Exists
      tolerationSeconds: 300
  status:
    phase: Pending
    qosClass: BestEffort
